### DISCUSSION TITLE
Sorting by publication year

### DISCUSSION THREAD

**Discussion by RandomWare** [advplyr/audiobookshelf#917] [Ideas]
It would be nice to be able to sort the audiobook library view by the book's publication year. It would help with complex ordered series, like series that are related or have multiple alternate reading orders, or with standalone books of an author.

**Comment by advplyr** [comment]
Agree, I have created a feature request from your discussion

**Comment by advplyr** [comment]
Added in [2.1.3](https://github.com/advplyr/audiobookshelf/releases/tag/v2.1.3)
